plugins/ohdear/api_token.go

```diff
@@ -0,0 +1,40 @@
+package ohdear
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIToken,
+		DocsURL:       sdk.URL("https://ohdear.app/docs/integrations/the-oh-dear-api#get-your-api-token"),
```

Comment:
From what I can see, this talks about the OhDear API.

I could only find this repo's README https://github.com/ohdearapp/ohdear-cli as a mention of the OhDear CLI, is there an official link we could use?

Reply:
It looks like the docs haven't yet been updated to mention that it's the official CLI. 🤔

But here's the current docs that mention it: https://ohdear.app/docs/integrations/3rd-party-integrations-of-oh-dear#cli-tool

Reply:
We've now updated the docs to have an official page!
https://ohdear.app/docs/integrations/our-cli-tool

Would that be ok to use?

Reply:
Let's use it in the DocsURL in `plugins/ohdear/ohdear.go`, for the CLI, rather than the credential 😄 

Thank you for your quick turnaround here!